airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3299,7 +3299,7 @@ components:
             - backfill
             - manual
             - scheduled
-            - dataset_triggered
+            - asset_triggered
```

Comment:
These probably need breaking change news fragments? And maybe a mention in the endpoint description? (Something like *Changed in 3.0.0: The asset_xxx value was renamed from dataset_xxx*)

Reply:
sure. just added